config.json

```diff
@@ -0,0 +1,3 @@
+{ 
```

Comment:
This file's title should be `configs.json` to work properly. Once it's named correctly, it should be automatically ignored by git (we don't want it version controlled)